packages/react-scripts/fixtures/kitchensink/src/features/env/NodePath.js

```diff
@@ -1,21 +1,27 @@
-import React from 'react'
+import React, { Component, PropTypes } from 'react'
 import load from 'absoluteLoad'
 
-export default class extends React.Component {
-  constructor(props) {
-    super(props);
+export default class extends Component {
+  static propTypes = {
+    notifyRendered: PropTypes.func
+  }
 
-    this.done = () => {};
-    this.props.setCallWhenDone && this.props.setCallWhenDone((done) => {
-      this.done = done;
-    });
+  static defaultProps = {
+    notifyRendered: () => {}
```

Comment:
It's not specified when running under Jest: https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/fixtures/kitchensink/src/features/syntax/ArrayDestructuring.test.js